src/components/accessories/patientTherapy/PatientTherapy.tsx

```diff
@@ -57,9 +62,13 @@ const PatientTherapy: FC<TProps> = ({
     }
   }, [activityTransitionState, createTherapyReset]);
 
+  useEffect(() => {
+    getTherapiesByPatientId(patientData?.code);
+  }, [patientData, getTherapiesByPatientId]);
+
   const onSubmit = (therapy: TherapyRowDTO) => {
     setShouldResetForm(false);
-    therapy.patID = patientId;
+    therapy.patID = patientData;
```

Comment:
patID should be renamed in patientData in order to be compliant between props name and parameter.

Reply:
patID depends on end point data model.

Reply:
So, is this a name error on the API? Should it be therapy.patData instead of patID?

Reply:
the property in api model is patID